src/App.js

```diff
@@ -8,10 +8,39 @@ function App() {
   return (
     <div className="App">
       <Form habits={habits} onSubmit={addHabit} />
-      <List habits={habits} onItemClick={toggleCompleted} />
+      <List
+        displayToday={displayToday}
+        habits={habits}
+        onItemClick={toggleCompleted}
+      />
     </div>
   )
 
+  function displayToday() {
```

Comment:
Maybe extract this function into a util.js file, then you can import it directly in `List.js`